js/web/lib/wasm/jsep/backend-webgpu.ts

```diff
@@ -207,9 +207,6 @@ export class WebGpuBackend {
     };
 
     Object.defineProperty(this.env.webgpu, 'device', {value: this.device});
-
-    // init queryType, which is necessary for createKernel
-    this.setQueryType();
```

Comment:
When we set "ort.env.webgpu.profiling = { mode: "default" };" before InferenceSession.create, this may impact some behaviors during session creation, like jsepCreateKernel. So the removal here might cause some regression.

Reply:
Done. Revert the change here.